.github/workflows/optimize-dependency-check.yml

```diff
@@ -0,0 +1,49 @@
+# TODO: Remove this pipeline once Optimize inherits from zeebe-parent
```

Comment:
Please add a ticket number here where this gets implemented, so that we don't overlook a TODO